### PR TITLE
Update Flow / minor fixes

### PR DESCRIPTION
**Pull request by CoreyAR**
# Description

I noticed that flow wasn't picking up all the error so I upgrade it and improved some types. Most notably I sealed state and prop types so we can catch extraneous props.

### CODE REVIEW

**Comment by combinatorist** [comment]
@CoreyAR is this PR still relevant in light of the new framework you wanted to use?

**Comment by CoreyAR** [comment]
Some changes look good. the pdf-document changes are not important. I'll take this and update.

File: .flowconfig [hunk 1]
```diff
@@ -34,3 +34,7 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
 suppress_comment=\\(.\\|\n\\)*\\$FlowExpectedError
 
 [strict]
+
+
+[version]
+^0.89.0
```
Comment (combinatorist) [inline .flowconfig]: Bummer - looks like this is specified here and in pakcage.json. Why does it need to be set here?

File: src/screens/poa-form/index.js [hunk 2]
```diff
@@ -25,30 +25,32 @@ type AddressKeysType =
   | 'region'
   | 'postal_code'
 
-type PoAFormProps = {
+type PoAFormProps = {|
```
Comment (combinatorist) [inline src/screens/poa-form/index.js]: Looks like this is the Flow way of enforcing an exact type: https://stackoverflow.com/a/48664988/5555637 :heavy_check_mark:

File: src/screens/poa-form/index.js [hunk 3]
```diff
@@ -25,30 +25,32 @@ type AddressKeysType =
...
-} & FormInputs
+  errors: FormInputErrors,
+  ...FormInputs
+|}
 
 class PoAForm extends Component<PoAFormProps, PoAFormState> {
   static navigationOptions = ({navigation}) => ({})
 
-  constructor(props) {
+  constructor(props: PoAFormProps) {
```
Comment (combinatorist) [inline src/screens/poa-form/index.js]: This looks like it is also enforcing type.

File: src/screens/poa-form/index.js (hunk 2 above)
Comment (CoreyAR) [reply src/screens/poa-form/index.js]: correct

File: src/screens/poa-form/index.js (hunk 3 above)
Comment (CoreyAR) [reply src/screens/poa-form/index.js]: correct

File: .flowconfig (hunk 1 above)
Comment (CoreyAR) [reply .flowconfig]: Flow could be installed locally or globally. It enforces the version being used to type the project regardless of where it is installed.

**Comment by CoreyAR** [comment]
That is correct. The PR originally made a fix to the old form. I removed that and updated the description because now it is only technical fixes.

File: .flowconfig (hunk 1 above)
Comment (combinatorist) [reply .flowconfig]: Goctha - thanks!

**Pull request merged**